tests/samples/scenarios.js

```diff
@@ -12,6 +12,9 @@ export const SCENARIODATA_WITH_USERS = {
   ...SHARED_SCENARIOS_LIST[0],
 };
 
-export const DEFAULT_SCENARIOS_LIST_DATA = {
-  ...DEFAULT_SCENARIOS_LIST,
-};
+export const DEFAULT_SCENARIOS_LIST_DATA = [...DEFAULT_SCENARIOS_LIST];
+
+export const DEFAULT_SCENARIOS_LIST_DATA_WITH_DEPTHS = [...DEFAULT_SCENARIOS_LIST].map((scenario) => ({
+  ...scenario,
+  depth: 0,
+}));
```

Comment:
I don't think we will ever reuse this array in other tests, because it is really specific to the scenario selector. We could probably move it and declare it directly in the file _src/components/CreateScenarioButton/CreateScenarioButton.spec.js_